src/download-helpers.js

```diff
@@ -132,12 +132,19 @@ export async function downloadReport(url, format, width, height, filename, authT
       }
     }
 
-    await browser.close();
-
     const timeCreated = time.valueOf();
     const data = { timeCreated, dataUrl: buffer.toString('base64'), };
-
     await readStreamToFile(data.dataUrl, filename, format);
+
+    if (transport !== undefined) {
+      let emailTemplateImageBuffer = await page.screenshot({
```

Comment:
Minor: I didn't see value of emailTemplateImageBuffer being changed after, we could use const

Reply:
Thanks! used const.